authentication/auth_method_client_cert.go

```diff
@@ -41,7 +41,11 @@ func (a servicePrincipalClientCertificateAuth) name() string {
 	return "Service Principal / Client Certificate"
 }
 
-func (a servicePrincipalClientCertificateAuth) getAuthorizationToken(sender autorest.Sender, oauthConfig *adal.OAuthConfig, endpoint string) (*autorest.BearerAuthorizer, error) {
+func (a servicePrincipalClientCertificateAuth) getAuthorizationToken(sender autorest.Sender, oauth *MultiOAuth, endpoint string) (autorest.Authorizer, error) {
+	if oauth.OAuth == nil {
+		return nil, fmt.Errorf("Error MultiOAuth did not contain a regular oauth token")
```

Comment:
this is an error seen by users; so it'd be better to make this more descriptive (e.g. "an oauth token wasn't configured correctly; please file a bug with more details")